src/Packetery/Module/CurrencySwitcherFacade.php

```diff
@@ -22,16 +22,28 @@ class CurrencySwitcherFacade {
 	public static $supportedCurrencySwitchers = [
```

Comment:
Zde prosím přidej do docu poznámku: Use the name from the settings export, optionally extended with the publisher name.